### PR TITLE
Changing .xml to .md

### LINKED ISSUE #380: Specifications are in markdown now
#### Describe the bug
The Core and Validation spec files are markdown now, not XML.

The links work fine, but the text still says XML.

#### How to Reproduce
https://json-schema.org/specification-links#latest-snapshot-(work-in-progress)

image

#### Expected behavior
Either remove the file extension or change to `.md`

### PR DESCRIPTION
**Pull request by DhairyaMajmudar**
**GitHub Issue:** #380 
closes #380  

**Summary**:

changed the .xml extension to .md extension for
1. jsonschema-core.md
2. jsonschema-validation.md 

**Files changed**
1.  `pages/specification-links.md`

**Screenshot Before**
image

**Screenshot After**
image

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

Yes/No
No

### CODE REVIEW

**Comment by DhairyaMajmudar** [comment]
@benjagm I have made the changes pls. have a look

**Review by DarhkVoyd — APPROVED** [review]
LGTM!

**Review by benjagm — APPROVED** [review]
Looks good to me. Great job!

File: pages/specification-links.md [hunk 1]
```diff
@@ -432,8 +432,8 @@ The next unreleased draft is a work in progress.  You can [give feedback and get
 
 The specification links here link to the XML and JSON sources.  We do not provide rendered [work-in-progress](work-in-progress) drafts except near the very end of a publication cycle, during the final review period.
```
Comment (gregsdennis) [inline pages/specification-links.md:433]: Can you change "XML and JSON" to just state "raw", please?

**Review by gregsdennis — APPROVED** [review]
One minor comment. Then good.

File: pages/specification-links.md (hunk 1 above)
Comment (DhairyaMajmudar) [reply pages/specification-links.md:433]: sure : )

**Review by gregsdennis — APPROVED** [review]
@DhairyaMajmudar I had already approved this. There's no need to re-request the approval.

**Pull request merged**